tools/Makefile.unix

```diff
@@ -506,42 +506,28 @@ pass2dep: context tools/mkdeps$(HOSTEXEEXT) tools/cnvwindeps$(HOSTEXEEXT)
 # location: https://bitbucket.org/nuttx/tools/downloads/.  See README.txt
 # file in the NuttX tools GIT repository for additional information.
 
-do_config: dirlinks apps_preconfig
+config: apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} kconfig-conf Kconfig
 
-config: do_config clean_context
-
-do_oldconfig: dirlinks apps_preconfig
+oldconfig: apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} kconfig-conf --oldconfig Kconfig
 
-oldconfig: do_oldconfig clean_context
-
-do_olddefconfig: dirlinks apps_preconfig
+olddefconfig: apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} kconfig-conf --olddefconfig Kconfig
 
-olddefconfig: do_olddefconfig clean_context
-
-do_menuconfig: dirlinks apps_preconfig
+menuconfig: apps_preconfig
 	$(Q) APPSDIR=${CONFIG_APPS_DIR} kconfig-mconf Kconfig
 
-menuconfig: do_menuconfig clean_context
```

Comment:
In apps `clean_context` deletes *.pdat and *.bdat files.  When removing an app through menuconfig for example, without `clean_context` the app won't be removed. 
And a `make` after a `clean` would fail with an undefined reference, since the `built_list/proto.h` are removed (by `clean`) then repopulated (during `depend`) with all the *.pdat and *.bdat file.
Is removing `clean_context` necessary in a parallel build scenario?

Reply:
Since './tools/configure.sh config' stage depends on the 'dirlinks' stage and do not depend on the 'context' stage.  From './tools/configure.sh config V=1', it show no  'context' built. So I think 'clean_context' may not necessary.

Reply:
@Ouss4  Sorry, just get your point. I'll think about it how to update the PR later.

Reply:
Done, please take a look. Thanks

Reply:
Thanks, Liu. It looks okay as far as the scenario I was talking about is concerned.
Does this fix your issue?